### PR TITLE
bunch of little text changes

### PR DESCRIPTION
**Pull request by krisl**

### CODE REVIEW

File: docs/tutorials/workflow.rst [hunk 1]
```diff
@@ -140,18 +140,18 @@ UML activity diagram
...
 
 Suspicious behavior
 ===================
 
-If your workflow doesn't match configure-once approach then it may be a
+If your workflow doesn't match the configure-once approach then it may be a
 symptom of wrongly written CMake code. Especially when you have to run
-``cmake -H. -B_builds`` twice or when ``cmake --build _builds`` doesn't catch
-updates from CMake code.
+``cmake -H. -B_builds`` twice or when ``cmake --build _builds`` doesn't detect
```
Comment (ruslo) [inline docs/tutorials/workflow.rst:153]: Updates come **from** CMake source code, not **to** CMake source code.

File: docs/tutorials/out-of-source.rst [hunk 2]
```diff
@@ -7,18 +7,18 @@ Out-of-source build
 ===================
 
 The next important term is "out-of-source build". "Out-of-source build" is a
-good practice of keeping separately generated files from
-:ref:`binary tree <binary tree>` and source files from
-:ref:`source tree <source tree>`. CMake does support contrary "in-source build"
-layout, but such an approach has no real benefits and not recommended.
+good practice of keeping separate, the generated files of the
```
Comment (ruslo) [inline docs/tutorials/out-of-source.rst:10]: The comma probably is not needed here.

File: docs/tutorials/out-of-source.rst [hunk 3]
```diff
@@ -84,14 +84,15 @@ compare it with similar file for in-source build:
...
-your project settings in IDE.
-
-Another note is that out-of-source mean not only set ``cmake -B_builds`` but
-also remember to put any kind of automatically generated files to ``_builds``.
-E.g. if you have C++ template ``myproject.h.in`` which is used to generate
-``myproject.h``, then you need to keep ``myproject.h.in`` in source tree and put
-``myproject.h`` to the binary tree.
+your project settings in your IDE.
+
+Another note is that using an out-of-source build means that  not only do you
```
Comment (ruslo) [inline docs/tutorials/out-of-source.rst:93]: `that  not` extra space here

**Review by ruslo — CHANGES_REQUESTED** [review]

File: docs/tutorials/workflow.rst (hunk 1 above)
Comment (krisl) [reply docs/tutorials/workflow.rst:153]: yes, but the updates have been made *to* the CMake source code, and those are the changes that are detected.
The original phrase was "catch updates from CMake code", hmmm if you really want to keep "from", ill have to think how to word this in English.
I changed it to "detect updates to the CMake code", kind of a short from of "detect updates that have been made to the CMake code", trying to keep the same casual writing style

File: docs/tutorials/out-of-source.rst (hunk 2 above)
Comment (krisl) [reply docs/tutorials/out-of-source.rst:10]: indeed, will fix

File: docs/tutorials/out-of-source.rst (hunk 3 above)
Comment (krisl) [reply docs/tutorials/out-of-source.rst]: done

File: docs/tutorials/workflow.rst (hunk 1 above)
Comment (ruslo) [reply docs/tutorials/workflow.rst:153]: > detect updates that have been made to the CMake code

Nice variant

Comment (krisl) [reply docs/tutorials/workflow.rst:153]: maybe "detect the changes coming from the CMake code".  But I advise against this, "from" is probably more "active" here than you want.  
I believe my original suggestion is in line with your intended writing style.
I'm assuming your native language is a slavic one, please let me know if I'm mistaken)

Comment (ruslo) [reply docs/tutorials/workflow.rst:153]: > detect the changes coming from the CMake code

Good for me too

> detect updates that have been made to the CMake code

as well as this one, can you choose from these two? :)

> I'm assuming your native language is a slavic one

Yep, Russian

Comment (krisl) [reply docs/tutorials/workflow.rst:153]: окей

**Review by ruslo — APPROVED** [review]
Looks good, thank you 👍

**Pull request merged**